test/unit/points/test-calculate-contracted-hours.js

```diff
@@ -0,0 +1,25 @@
+const calculateContractedHours = require('../../../app/points/calculate-contracted-hours')
+const DefaultContractedHours = require('../../../app/points/domain/default-contracted-hours')
+const expect = require('chai').expect
+
+describe('points/calculate-contracted-hours', function () {
+  var contractedHoursPerWeek
+  var defaultContractedHours
+  var offenderManagerTypeId
+
+  beforeEach(function () {
+    contractedHoursPerWeek = 7
+    defaultContractedHours = new DefaultContractedHours(12, 7)
+  })
+
+  it('should return null when unsupported type is selected', function () {
+    offenderManagerTypeId = 4
+    /* eslint-disable no-unused-expressions */
+    expect(calculateContractedHours(contractedHoursPerWeek, defaultContractedHours, offenderManagerTypeId)).to.be.null
+  })
+
+  it('should return contracted hours', function () {
+    offenderManagerTypeId = 1
+    expect(calculateContractedHours(contractedHoursPerWeek, defaultContractedHours, offenderManagerTypeId)).to.be.an('object')
+  })
```

Comment:
Can you add a few tests which check concrete values? For example, when passed an invalid contracted hours then the base is set to default, and one where base is consistent with contractedHoursPerWeek?

(Some of the tests from `test-calculate-available-points` might be better placed in this test class)